.travis.yml

```diff
@@ -1,21 +1,24 @@
 language: php
 sudo: false
 
+services:
```

Comment:
Do we actually need these services running? I don't think we actually hit the DB.

Reply:
It's required if we use the matrix values here:

  matrix:
    - DB=pgsql
    - DB=mysqli

And those are required by the moodle-plugin-ci

Reply:
Okay no worries.